.lando.base.yml

```diff
@@ -40,6 +40,10 @@ tooling:
     service: appserver
     description: Runs composer update drupal/core webflo/drupal-core-require-dev --with-dependencies
     cmd: composer update drupal/core webflo/drupal-core-require-dev --with-dependencies
+  dsi:
+    service: appserver
+    description: Runs drush site install with drupal8 recipe's --db-url
+    cmd: /app/vendor/bin/drush si --db-url=mysql://drupal8:drupal8@database:3306/drupal8
```

Comment:
Do we really need to provide the database credentials? These should already be provided via the lando.settings.php.

In most cases people where people need to install drupal, they will either use config_installer, or if they are starting from scratch they should make a conscious choice of installation profile.

Reply:
Removed wrapper commands.